lib/workers/repository/process/lookup/filter-checks.ts

```diff
@@ -64,27 +66,82 @@ export async function filterInternalChecks(
       }
       candidateRelease = updatedCandidateRelease;
 
+      const candidateVersionsWithoutReleaseTimestamp: Record<
+        MinimumReleaseAgeTimestamp,
+        string[]
+      > = {
+        required: [],
+        optional: [],
+      };
+
       // Now check for a minimumReleaseAge config
       const { minimumConfidence, minimumReleaseAge, updateType } =
         releaseConfig;
-      if (
-        is.nonEmptyString(minimumReleaseAge) &&
-        candidateRelease.releaseTimestamp
-      ) {
+      if (is.nonEmptyString(minimumReleaseAge)) {
+        let minimumReleaseAgeTimestamp: MinimumReleaseAgeTimestamp = 'optional';
```

Comment:
What's the best way of getting this to stay in sync with the config value's `default`?